pkg/docker/index.html

```diff
@@ -152,7 +152,10 @@ <h1 translatable="yes">Can&rsquo;t connect to Docker</h1>
           </tr>
           <tr>
             <td translate>Image:</td>
-            <td colspan="3" id="container-details-image"></td>
+            <td colspan="3">
+            <div id="container-details-image"></div>
+            <div id="container-details-image-id"></div>
```

Comment:
Ah, so this will *always* show the hash, and only the name when available? I was referring to [the screenshot in the bug report](https://camo.githubusercontent.com/3abc0f6e142923e8b3219dd84d68c2858cae9b5a/687474703a2f2f636f636b7069742d70726f6a6563742e6f72672f696d616765732f73637265656e73686f742d646f636b65722e706e67) which showed only the name, but showing the ID as well makes sense. I wasn't saying it's wrong, I was just wondering how you intended it to look like.